src/popup/popup.html

```diff
@@ -5,6 +5,7 @@
   <title>Superhero Wallet</title>
   <link rel="stylesheet" href="<%= PLATFORM === 'web' ? '/' : '' %>popup.css">
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no, viewport-fit=cover" />
+  <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Sans:300,400,500,600,700&display=swap" rel="stylesheet">
```

Comment:
Just wondering if we should bundle the font in the extension package for performance reasons, as well as centralization concerns?

Reply:
Yeah, please do not use googleapis.

Reply:
You can see how it is included in the superhero-ui [here](https://github.com/aeternity/superhero-ui/blob/fb2e17536b427dd6498fb15c6075071ef3ed6dfd/src/styles/_fonts.scss) and [here](https://github.com/aeternity/superhero-ui/blob/122adac107f3a92807ede809676a88763f929e81/package.json#L29) and use it in the same way.